.github/ISSUE_TEMPLATE/design_call.md

```diff
@@ -11,7 +11,8 @@ assignees: ''
 Date: 2020-MM-DD
 Time: 4:00pm
 Timezone: UTC
```

Comment:
No need for Timezone if we're using UTC now

```suggestion
```

Reply:
I'd still leave it in as a human readable time zone identifier.

How I see it is that people can write time and date whoever they want in the Date, Time and Timezone fields. But UTCTime has to follow the specific format the script understands. What do you think?

Reply:
Yeh I'd be in favour of keeping them for readability sake.